Guardfile

```diff
@@ -0,0 +1,70 @@
+# A sample Guardfile
```

Comment:
This was a generated file, should I remove this line? I was working with this guardfile for weeks and it's good-enough for MIQ.